code/modules/events/meteor_wave.dm

```diff
@@ -47,8 +47,7 @@
 			kill()
 
 /datum/round_event/meteor_wave/announce(fake)
-	//priority_announce("Meteors have been detected on collision course with the station.", "Meteor Alert", 'sound/ai/meteors.ogg') //ORIGINAL
-	priority_announce("Meteors have been detected on collision course with the station. Estimated time until impact: [round((startWhen * SSevents.wait) / 10, 0.1)] seconds.", "Meteor Alert", ANNOUNCER_METEORS) //ORIGINAL
+	priority_announce("Meteors have been detected on collision course with the station.", "Meteor Alert", ANNOUNCER_METEORS)
```

Comment:
The announcement saying how long they'll take to come was a good change, we shouldn't revert that.